fake/Cargo.toml

```diff
@@ -59,6 +60,7 @@ bigdecimal = ["bigdecimal-rs", "rust_decimal"]
 geo = ["geo-types", "num-traits"]
 http = ["dep:http", "url-escape"]
 bson_oid = ["bson"]
+clap = ["dep:clap"]
```

Comment:
the feature name should be "cli"?,  `cli = ["dep:clap"]`

Reply:
Done